ntoskrnl/mm/ARM3/virtual.c

```diff
@@ -375,9 +375,7 @@ MiDeletePte(IN PMMPTE PointerPte,
     MMPTE TempPte;
     PFN_NUMBER PageFrameIndex;
     PMMPDE PointerPde;
-
-    /* PFN lock must be held */
-    MI_ASSERT_PFN_LOCK_HELD();
```

Comment:
Assert the lock not held?

Reply:
MiAcquirePfnLock does this for itself, now that recursive spinlock acquisition is forbidden everywhere.